src/ucx_plugin.c

```diff
@@ -888,58 +920,19 @@ ncclNet_v5_t ucxPlugin_v5 = {
   .closeListen = nccl_ucx_close_listen
 };
 
-static ncclResult_t nccl_ucx_get_properties_v4(int dev, ncclNetProperties_v4_t* props) {
-  ncclNetProperties_v6_t props_v6;
-  ncclResult_t ret = nccl_ucx_get_properties(dev, &props_v6);
-  if (ret != ncclSuccess) return ret;
-  props->name = props_v6.name;
-  props->pciPath = props_v6.pciPath;
-  props->guid = props_v6.guid;
-  props->ptrSupport = props_v6.ptrSupport;
-  props->speed = props_v6.speed;
-  props->port = props_v6.port;
-  props->maxComms = props_v6.maxComms;
-  return ncclSuccess;
-};
-
-static ncclResult_t nccl_ucx_isend_v4(void *sendComm, void* data, int size, void *mhandle, void** request) {
-  return nccl_ucx_isend(sendComm, data, size, 0, mhandle, request);
-}
-static ncclResult_t nccl_ucx_irecv_v4(void* recvComm, void* data, int size, void* mhandle, void** request) {
-  int tag = 0;
-  return nccl_ucx_irecv(recvComm, 1, &data, &size, &tag, &mhandle, request);
-}
-static ncclResult_t nccl_ucx_iflush_v4(void* recvComm, void* data, int size, void* mhandle, void** request) {
-  return nccl_ucx_iflush(recvComm, 1, &data, &size, &mhandle, request);
-}
-static ncclResult_t nccl_ucx_connect_v4(int dev, void* handle, void** sendComm) {
-  ncclResult_t ret;
-  do {
-    ret = nccl_ucx_connect(dev, handle, sendComm);
-  } while (ret == ncclSuccess && *sendComm == NULL);
-  return ret;
-}
-static ncclResult_t nccl_ucx_accept_v4(void* listenComm, void** recvComm) {
-  ncclResult_t ret;
-  do {
-    ret = nccl_ucx_accept(listenComm, recvComm);
-  } while (ret == ncclSuccess && *recvComm == NULL);
-  return ret;
-}
-
-ncclNet_v4_t ucxPlugin_v4 = {
+ncclNet_v5_t ucxPlugin_v5 = {
   .name = "UCX",
   .init = nccl_ucx_init,
   .devices = nccl_ucx_devices,
-  .getProperties = nccl_ucx_get_properties_v4,
+  .getProperties = nccl_ucx_get_properties_v6,
```

Comment:
Should this be v5 or are you reusing v6 for v5 when it hasn't changed?

Reply:
reusing